modules/tensor_mechanics/src/utils/ElasticityTensorTools.C

```diff
@@ -108,4 +108,26 @@ momentJacobianWC(const RankFourTensor & r4t, unsigned int i, unsigned int k, Rea
 
   return test * phi * sum;
 }
+
+Real
+getIsotropicShearModulus(const RankFourTensor & elasticity_tensor)
+{
+  const Real shear_modulus = elasticity_tensor(0, 1, 0, 1);
+  if (shear_modulus != elasticity_tensor(0, 2, 0, 2))
+    mooseError("Check to ensure that your Elasticity Tensor is truly Isotropic");
```

Comment:
The correct isotropy check is much more complicated and it is implemented in R4T. Do we want to keep this hack here?